### ISSUE TITLE
Asterix shown even if the field is not required (fact sheets)

### ISSUE THREAD

**Issue by Didayolo** [codalab/codabench#755]
In this example, the field "Fact sheets" is not required.

<img width="589" alt="Capture d’écran 2022-11-30 à 15 03 17" src="https://user-images.githubusercontent.com/11784999/204816261-e4d8b601-93ce-4e45-8f83-4283413cd2c1.png">

**Comment by nilayrajderkar** [comment]
I would like to take this up

**Comment by Didayolo** [comment]
@nilayrajderkar 

What do you mean? Do you wish to correct the problem and submit a pull request?

**Comment by nilayrajderkar** [comment]
Yes, that is correct. I would like to take up the issue and submit a pr

**Comment by Didayolo** [comment]
Great news. You are welcome to do so.
